### PR TITLE
User stories

### PR DESCRIPTION
**Pull request by jmcarrillog**
Commiting first version of user stories for the minimum viable product.

### CODE REVIEW

File: docs/user-stories.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Description of users' roles and basic user stories
+
+## Basic set of users
+* Pacient
```
Comment (gaberch) [inline docs/user-stories.md]: *Patient

File: docs/user-stories.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+## Common user stories
+
+### System administrator
+
+1. Logs into the system through a web app using proper credentials.
+2. Creates an account for a patient.
+3. Creates an account for a doctor.
+4. Deletes a patient account.
+5. Deletes a doctor account.
+
```
Comment (gaberch) [inline docs/user-stories.md]: 6. Grants appropriate level of privileges to patient and doctor accounts.

File: docs/user-stories.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+### Doctor
+
+1. Log into the system through a web app using proper credentials.
+2. Look for one of the assigned patient’s by his/her last name.
+3. For a particular patient, see the details of his/her last measure.
+4. For a particular patient, see summary statistics of the last week including a plot.
```
Comment (gaberch) [inline docs/user-stories.md]: Maybe for points 4 & 5 we can combine to say: For a particular patient, see summary statistics over a period of time chosen by doctor (e.g. dd/mm/yyyy to dd/mm/yyyy)

File: docs/user-stories.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+7. See the rank of all the assigned patients sorted by their average measure over last week.
+8. See the rank of all the assigned patients sorted by their average measure over last month.
+
+### Patient
+
+1. Log into the system through a mobile app using proper credentials.
+2. Write and upload a measure of his/her blood pressure.
+3. Delete a previous measure of his/her blood pressure.
+4. See the details of the last measure.
+5. See summary statistics of the last week including a plot.
```
Comment (gaberch) [inline docs/user-stories.md]: Same comment as for doctor user story. Pull data based on request of dd/mm/yyyy to dd/mm/yyyy

File: docs/user-stories.md (hunk 2 above)
Comment (gaberch) [inline docs/user-stories.md]: 6. Grants appropriate level of privileges to patient and doctor accounts.

**Comment by adkulas** [comment]
closes #15

**Pull request merged**